pac4j-jwt/src/main/java/org/pac4j/jwt/credentials/authenticator/JwtAuthenticator.java

```diff
@@ -138,9 +140,19 @@ private static void createJwtProfile(final TokenCredentials credentials, final S
             subject = JwtProfile.class.getSimpleName() + UserProfile.SEPARATOR + subject;
         }
 
-        Map<String, Object> claims = new HashMap<>(claimSet.getClaims());
-        claims.remove(JwtConstants.SUBJECT);
-        final UserProfile profile = ProfileHelper.buildProfile(subject, claims);
+        final Map<String, Object> attributes = new HashMap<>(claimSet.getClaims());
+        attributes.remove(JwtConstants.SUBJECT);
+        final List<String> roles = (List<String>) attributes.get(JwtGenerator.INTERNAL_ROLES);
+        attributes.remove(JwtGenerator.INTERNAL_ROLES);
+        final List<String> permissions = (List<String>) attributes.get(JwtGenerator.INTERNAL_PERMISSIONS);
```

Comment:
Why remove them? What's the harm in keeping them both as roles/permissions and as attributes?